### DISCUSSION TITLE
[Node Dependencies] Hidden Terminal

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [Idered/iridium#32] [Ideas]
Hey! Excellent extension! I'm already using it.

Here's an idea, when starting the terminal to execute the commands, can it be hidden? I do this all the time either with CMD or Powershell in my own Python/Electron Windows desktop apps. That black terminal window without text that popups, it's obviously annoying, but can cause fear in the least experienced.

Wishing you health, peace, and success.

**Comment by angelhdzmultimedia** [comment]
Was reading this: https://stackoverflow.com/questions/51338738/nodejs-hide-window-from-spawned-bat-file-process-windows

**Reply by angelhdzmultimedia** [reply]
Oh I see you're using https://www.npmjs.com/package/cross-spawn

**Comment by angelhdzmultimedia** [comment]
Any updates?

**Comment by Idered** [comment]
fixed
